tests/cursor/bug0920-001.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+PHPC-924: Using a projection with an empty field name causes a crash when destroying cursor
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE) ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(STANDALONE);
+
+$query = new MongoDB\Driver\Query([], ['projection' => ['' => 1]]);
+
+echo throws(function() use ($manager, $query) {
+    $cursor = $manager->executeQuery(NS, $query);
```

Comment:
To clarify, we are currently limited to a RuntimeException when `executeQuery()` is called, because `_mongoc_cursor_new_with_opts()` reports an error with the `MONGOC_ERROR_CURSOR` domain and `MONGOC_ERROR_CURSOR_INVALID_CURSOR` code.

Allowing us to throw an InvalidArgumentException at Query construct time should require less than ten lines of code. I'll create a ticket and submit a PR soon.